src/css/flags/ads.less

```diff
@@ -60,7 +60,13 @@
     }
 }
 
-.jwplayer.jw-flag-ads-vpaid {
+.jwplayer.jw-flag-ads.jw-flag-touch {
+    .jw-controlbar {
+        display: table;
+    }
+}
+
+.jwplayer.jw-flag-ads-vpaid, .jwplayer.jw-flag-touch.jw-flag-ads-vpaid {
```

Comment:
Yeah, for the scope of this issue we're only concerned about full slot ads, which are VPAID.